README.md

```diff
@@ -46,7 +46,7 @@ You can also **limit the applications Webpack builds** with `--env.entry`:
 yarn watch --env.entry static-pages,auth
 ```
 
-The `entryname` for your application can be found in its `manifest.json` file.
+The `entry name` for your application can be found in its `manifest.json` file.
```

Comment:
```suggestion
The `entryName` for your application can be found in its `manifest.json` file.
```
I think this references the actual property name in the json for example:
https://github.com/department-of-veterans-affairs/vets-website/blob/5a43e57c29f8e73f4bfa2eb8e6e7020c1b687cbd/src/applications/burials/manifest.json#L4